lib/config/config.js

```diff
@@ -219,6 +219,24 @@ module.exports = function (fs, path, url, convict) {
         default: false,
         env: 'REPUTATION_SERVICE_ENABLE'
       },
+      enable_check: {
```

Comment:
style: these new props need to be camelCase, like other config props in this file.